### ISSUE TITLE
Computer freezes when train for around 2 hours

### ISSUE THREAD

**Issue by pienrojp** [microsoft/AirSim#999]
Hi, I am new to Airsim and game Engine.

I want to train my RL algorithm on a quadrotor but it seems like after 2hrs of training on Airsim my computer always freeze. I am running airsim on Ubuntu 16.04 with "NoDisplay" mode. 
My initial guess would be it has to be something with the resources and memory (?).

Is this an issue that happened only because of my computer or are there any way I can run my simulation longer (like overnight) with more efficient on resource usage?

Thank you very much!

**Comment by pienrojp** [comment]
Ok, I already found out that every time I run the simulation, the environment creates some log file inside ./Saved/Logs/ . I deleted them and now my simulation runs without any freezing.

**Comment by pienrojp** [comment]
Just an update.
I found a way to stop Unreal engine dumping logs by adding 

[Core.log]
global=off

to defaultEngine.ini and now my simulation is able to run as long as I want without memory issue.

**Comment by asimhirji** [comment]
Hi,

I am currently training a model in the landscape environment. After a certain amount of time/certain amount of episodes of training, the environment freezes where the rotors on the multirotor are misplaced and the only way to continue training is to press the back button to manually reset the simulation. I've tried running Landscape Mountains through creating a new unreal engine project as well as using the Landscape Mountains environment provided in the AirSim GitHub pages. However, both seem to cause the same issues described before. I've also written an automatic reset when no movement is taken, but that hasn't fixed the issue. I also tried deleting the log files and tried to make sure log files don't get added, but I haven't found a solution yet. Any help will be appreciated!
